pyproject.toml

```diff
@@ -67,6 +67,7 @@ fastapi = "~0.75.0"
 uvicorn = { extras = ["standard"], version = "~0.17.5" }
 semver = "^2.13.0"
 markupsafe = "1.1.1"
+nbconvert = "6.4.4"
```

Comment:
So this was because the `pip install .` to install zenml failed inside the docker images. I'm not sure if this is just a problem when installing it from the source files, or if we will run into the same issues once we released our pypi package, wdyt?

Reply:
Maybe we should try a `poetry build` to build a wheel and then try to install that with pip in a fresh virtualenv to see if we still run into conflicts?

Reply:
Was gonna suggest this as well. I could give it a go